Textfield v1 updates

samfloored opened the pull request:
@emma-floored 

Things here:
1. I added `header-text-fields` as a second class to apply to a text field. Happy to do this another way! It really mostly just clears out styles.
2. I moved text field helper messages to underneath the input like how errors are. Also moved the class to typography, where errors are.

<img width="550" alt="screen shot 2019-02-15 at 5 39 30 pm" src="https://user-images.githubusercontent.com/13873026/52888212-defc6500-3148-11e9-90ba-f794519d814d.png">
<img width="595" alt="screen shot 2019-02-15 at 5 39 20 pm" src="https://user-images.githubusercontent.com/13873026/52888213-defc6500-3148-11e9-98fa-1826d2321ae2.png">

Note: I think the colors aren't 100% final, still waiting on some design wisdom next week!

emma-floored commented on packages/blocks-base/styles/text-fields.styl:
```diff
@@ -5,14 +5,17 @@ $text-field-date-icon-color = $primary-00;
 
+$header-text-field-background-color = $text-field-background-color;
+$header-text-field-invis-border = 1px solid transparent;
+$header-text-field-border = 1px solid #ced4d2;
```
oh I see, this is the new color you mentioned? the other 2 examples didn't look right?

samfloored replied:
Yep! Sounds like it'll probably be `light-01`.

emma-floored commented on packages/blocks-base/styles/typography.styl:
```diff
@@ -235,3 +235,11 @@ h5, .blx-h5 {
+  blxUISmall()
+  color: $light-02;
+  position: absolute;
+  top: 100%;
```
just curious - what does this `top: 100%` do?

samfloored replied:
It makes it show up underneath its parent (abs pos child of a rel pos parent, so the `100%` refers to the parent height) I think.

emma-floored commented:
https://app.clubhouse.io/floored/story/13688/update-major-text-inputs

emma-floored commented on packages/blocks-docs/_includes/components/textfields.html:
```diff
@@ -124,7 +124,7 @@
     <div class="blx-text-field-container">
       <input type="text" value="Text input">
-      <span class="blx-text-field-helper">34/150</span>
+      <span class="blx-input-helper-message">34/150</span>
```
might need to rename this class in the React component as well

samfloored replied:
Is it possible it's not in any React component? Should I add something for it?

samfloored commented:
@emma-floored I checked with Liz, and it sounds like we want the default (`light-01`) border. Updated, rebased, and fpushed. Any other thoughts on this?
